### PR TITLE
Update details about Claude Sonnet 3.5 Model for Pro

### PR DESCRIPTION
**Pull request by MaedahBatool**
This PR updates the following:

- Adds mention of Claude Sonnet 3.5 as another supported LLM model for Chat and Commands for Cody Pro VS Code users
- Updates our Supported LLMs docs 
- Mentions that Claude Sonnet 3.5 is the default model for Chat and Commands for Cody Free users

### Reference:

- https://sourcegraph.slack.com/archives/C05AGQYD528/p1718896254676939

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@abeatrix @kalanchan would you please review. 🙏

File: docs/cody/capabilities/supported-models.mdx [hunk 1]
```diff
@@ -12,6 +12,7 @@ Cody supports a variety of cutting edge large language models for use in Chat an
 | OpenAI       | [gpt-4o](https://platform.openai.com/docs/models/gpt-4o) | -              | ✅              | ✅              |
 | Anthropic    | [claude-3 Haiku](https://docs.anthropic.com/claude/docs/models-overview#model-comparison)                                                     | -              | ✅              | ✅              |
 | Anthropic    | [claude-3 Sonnet](https://docs.anthropic.com/claude/docs/models-overview#model-comparison)                                                    | ✅              | ✅              | ✅              |
+| Anthropic    | [claude-3.5 Sonnet](https://docs.anthropic.com/claude/docs/models-overview#model-comparison)                                                    | ✅              | ✅              | -              |
```
Comment (abeatrix) [inline docs/cody/capabilities/supported-models.mdx:15]: Looks good! FYI Enterprise support should be ready in the next Sourcegraph release too 😄

File: docs/cody/clients/install-vscode.mdx [hunk 2]
```diff
@@ -311,9 +311,9 @@ Cody also works with Cursor, Gitpod, IDX, and other similar VS Code forks. To ac
 
 ## Supported LLM models
 
-Claude 3 Sonnet is the default LLM model for Cody Free users for Chat and Commands. Users on Cody **Pro** can choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Haiku, Opus and Sonnet), ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, GPT-4o, Mixtral, Google Gemini 1.5 Pro and Google Gemini 1.5 Flash.
+Claude Sonnet 3.5 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody **Pro** can choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Haiku, Opus and Sonnet), Claude 3.5 Sonnet, ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, GPT-4o, Mixtral, Google Gemini 1.5 Pro and Google Gemini 1.5 Flash.
```
Comment (abeatrix) [inline docs/cody/clients/install-vscode.mdx:314]:
```suggestion
Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody **Pro** can choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Haiku, Opus and Sonnet), Claude 3.5 Sonnet, ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, GPT-4o, Mixtral, Google Gemini 1.5 Pro and Google Gemini 1.5 Flash.
```

Sonnet 3 is still the default! Right now we are adding 3.5 Sonnet as an option!

**Review by abeatrix — APPROVED** [review]
Left a comment in line. Sonnet 3 is currently still the default model but other than that, lgtm!

**Pull request merged**